src/adapters/markdown.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * decaffeinate suggestions:
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+// This is our Markdown parser implementation
+
+const {renderHtml} = require('blueprint-markdown-renderer');
+
+const parseMarkdown = function(markdown, params, cb) {
+  // do not mutate passed-in argument "params", create our own options
+  const options = {
+    sanitize: (params != null ? params.sanitize : undefined)
+  };
+
+  // sanitize is enabled by default
```

Comment:
I don't think the below code matches this comment. `sanitize: (params != null ? params.sanitize : undefined)` is set to undefined (if null) then below compared to `null` so default wouldn't be enabled?